### PR TITLE
fix(images): fix second pasted image replaces the first one

### LINKED ISSUE #9966: Can't paste few images in one input using clipboard buffer
# Bug Report

## Description

As a user, I want to have the possibility to paste more than one image from the clipboard buffer

## Steps to reproduce

1. Open any chat
2. Add an image to the clipboard buffer (for example the screenshot_1)
3. Via control+v paste the screenshot_1 from the buffer
4. Add another image to the clipboard buffer (for example the screenshot_2)
5. Via control+v paste the screenshot_2 from the buffer

#### Expected behavior

2 images are added to the msg

#### Actual behavior

screenshot_1 was replaced by screenshot_2

https://user-images.githubusercontent.com/117639195/226622593-57f3d18b-65d3-4da4-9b6f-7a04db293fc1.mov

### Additional Information

- Status desktop version: 0.10.0-rc.13
- Operating System: macos

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #9966

Uses the validate function to paste images so that it concatenates the previous images.

Fixes the validate function to also accept data images.

Moves the size validation function to Utils to reuse the data path prefix constant and fix the possible crash when we try to get the size of a data image.

### CODE REVIEW

File: ui/imports/utils/Utils.qml [hunk 1]
```diff
@@ -324,6 +326,14 @@ QtObject {
         return Constants.acceptedDragNDropImageExtensions.some(ext => url.toLowerCase().includes(ext))
```
[the remark this answers is not in the document]
Comment (caybro) [reply ui/imports/utils/Utils.qml:326]: wdyt @jrainville ? I'd rather see this split into: `endsWith(ext) || startsWith(dataImagePrefix)`; imagine searching through the entire image blob for these things

[the remark this answers is not in the document]
Comment (jrainville) [reply ui/imports/utils/Utils.qml]: Fixed

**Review by caybro — CHANGES_REQUESTED** [review]

File: ui/imports/shared/status/StatusChatInput.qml [hunk 2]
```diff
@@ -883,11 +883,22 @@ Rectangle {
...
         isImage = true;
-
         imageArea.imageSource = imagePathsOrData
         control.fileUrlsAndSources = imageArea.imageSource
     }
 
+    // Use this to validate and show the images. The concatanation of previous selected images is done automatically
+    // Returns true if the images were valid and added
+    function validateImagesAndShowImageArea(imagePaths) {
+        const validImages = validateImages(imagePaths)
```
[the remark this answers is not in the document]
Comment (jrainville) [reply ui/imports/shared/status/StatusChatInput.qml:893]: It's an already existing function. Line 869

[the remark this answers is not in the document]
Comment (caybro) [reply ui/imports/shared/status/StatusChatInput.qml:893]: Ah oops, my bad

**Review by caybro — APPROVED** [review]
LGTM, thanks!

**Pull request merged**